File: SmallestElementArray/SmallestElement.cs
using System;

class Program {
    static void Main() {
        Console.Write("Enter number: ");
        int n = int.Parse(Console.ReadLine());

        Console.Write("Enter values: ");
        string[] input = Console.ReadLine().Split();
        int[] arr = Array.ConvertAll(input, int.Parse);

        int minimum = arr[0];
        for (int i = 1; i < n; i++) {
            if (arr[i] < minimum) {
                minimum = arr[i];
            }
        }

        Console.WriteLine($"Minimum number is {minimum}");
    }
}

File: SmallestElementArray/SmallestElement.js
const n = parseInt(prompt("Enter number: "), 10);
const arr = prompt("Enter values: ").split(' ').map(Number);

let minimum = arr[0];
for (let i = 1; i < n; i++) {
    if (arr[i] < minimum) {
        minimum = arr[i];
    }
}

console.log(`Minimum number is ${minimum}`);
